tests/ci_build/conda_env/cpu_test.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - pyyaml
 - cpplint
 - pylint
-- numpy
+- numpy=1.19.4
```

Comment:
This is a workaround, as the latest `shap` package appears to break with NumPy 1.19.5:
https://xgboost-ci.net/blue/organizations/jenkins/xgboost/detail/PR-6656/4/pipeline/221
```
     try:
>       from .. import _cext
E    ImportError: numpy.core.multiarray failed to import
```

Reply:
@trivialfis Have you run into this issue? What's your take on it?

Reply:
never seen it before.

> What's your take on it?

@RAMitchell worked on the c extension before so maybe have better insight.

Reply:
Using `shap` from conda-forge also fixes the issue, but unfortunately conda-forge has an outdated version of `shap`.